lib/rbslicer.rb

```diff
@@ -84,85 +84,80 @@ def wrapper_test()
     end
     return base_url
   end
-  # Public: Create field in Slicing Dice API
+  # Public: Create column in Slicing Dice API
```

Comment:
Change `Slicing Dice` to `SlicingDice`.
There are others occurrences in the code